tcl/logbuf_tcl.tcl

```diff
@@ -40,6 +40,12 @@ proc logbuf_string {var argn d} {
  dict append lb buf [binary format "a*c" $d 0]
 }
 
+proc logbuf_u8string {var argn d} {
```

Comment:
I opt for logbuf_unicode as the name suggests what's being passed as an argument.

Reply:
+